packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js

```diff
@@ -1,13 +1,15 @@
 import moment from 'moment-timezone';
 import {
+  unixTimestampToMoment,
   formatUnixTimestamp,
   isValidDateTime,
-  showRelativeTime,
+  getDateTime,
 } from './timestampUtilities';
 import timestampGenerator from './helpers/testHelpers';
 
 const timezone = 'Europe/London';
 const timestamp = 1539969006000; // 19 October 2018
+const locale = 'en-gb';
```

Comment:
I was as well but I assumed it's for a reason because it was explicitly declared before I refactored [(previous code)](https://github.com/bbc/psammead/blob/0eac2370cf1200bcdfdddc70f1a11f2c9b2b58c3/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L38). So to keep it consistent and not break code elsewhere I kept it in.

@jblaut What's the reason behind not localising datetime?

Reply:
https://github.com/bbc/psammead/blob/0eac2370cf1200bcdfdddc70f1a11f2c9b2b58c3/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L38

what the line above does is it sets the locale to `en-gb`, unless another locale is provided when the function is called. Basically instead of having to pass in the locale and having to set it to `en-gb` every single time, the locale only has to be passed in when we want it to be anything other than that.

Reply:
Wouldn't it make sense for `getDateTime()` to do this too? The line I marked in review is only necessary, because it doesn't set an internal default like `formatUnixTimestamp()`.

Reply:
@jblaut Thanks for the context, are you aware of any code that is relying on that locale defaulting behaviour? It may have made sense when we were only building articles for News, but I'd have expected that now the locale should be explicitly supplied everywhere it's needed, so I'd be interested to know if we can remove this default (and fix any bugs that arise due to imprecise assumptions elsewhere in the code). What do others think?

Reply:
If the requirement is to ensure that the `datetime` attribute output is in en-gb can we not explicitly set that when generating the value for that attribute, rather than relying on a default locale?